### PR TITLE
feat/PRO-2403/tile-generic-banner

### PR DESCRIPTION
**Pull request by RanaBug**
## Description

- FE implementation of the Generic Banner Tile component

## How Has This Been Tested?

- Unit test

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/apps/pillarx-app/index.tsx [hunk 1]
```diff
@@ -9,14 +9,19 @@ import { Projection } from '../../types/api';
...
+// utils
+import { componentMap } from './utils/configComponent';
 
 // components
-import { useGetWaitlistQuery } from '../../services/pillarXApiWaitlist';
-import PillarXLogo from './components/PillarXLogo/PillarXLogo';
 import SkeletonTiles from './components/SkeletonTile/SkeletonTile';
 import H1 from './components/Typography/H1';
+
+
```
Comment (IAmKio) [inline src/apps/pillarx-app/index.tsx:21]: Just a small thing:

```suggestion

```

...for consistency. I know we need to get Prettier in place, hoping to fit that in this sprint

File: src/apps/pillarx-app/index.tsx [hunk 2]
```diff
@@ -50,7 +55,7 @@ const App = () => {
 
 
 
-const displayAllTiles = () => {
+const DisplayAllTiles = () => {
```
Comment (IAmKio) [inline src/apps/pillarx-app/index.tsx:58]: Thank you 👍

File: src/apps/pillarx-app/images/banner-example.png
Comment (IAmKio) [inline src/apps/pillarx-app/images/banner-example.png:1]: Just wondering is this needed?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a small nitpick and a question, other than that all good!

File: src/apps/pillarx-app/images/banner-example.png
Comment (RanaBug) [reply src/apps/pillarx-app/images/banner-example.png:1]: Not really... I hesitated to delete it, let me remove it

File: src/apps/pillarx-app/index.tsx (hunk 1 above)
Comment (RanaBug) [reply src/apps/pillarx-app/index.tsx:21]: done

**Review by IAmKio — APPROVED** [review]
LGTM - another tile in the bag! 🚀

**Pull request merged**